apps/aws-notebook/start_jupyter_notebook.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+ECS_INSTANCE_STATE="/run/sandbox/fs/resources/aws-ecs-service/state"
+JUPYTER_KERNEL_DIR="/usr/local/share/jupyter/kernels/ECS IPython Kernel"
+JUPYTER_KERNEL_FILE=$JUPYTER_KERNEL_DIR"/kernel.json"
+SSH_PORT=22
+
+function fatal() {
+  echo "$@" >&2
+  exit 1
+}
+
+function ecs_ip() {
+    local ip_addr="$(jq -cMr .task_private_ip.value $ECS_INSTANCE_STATE 2>/dev/null || true)"
+    [[ -n "$ip_addr" ]] && echo "$ip_addr"
```

Comment:
Have you tested? This has problem. Please use the command exactly from my previous comment.

Here are the issues:

1. If `.task_private_ip.value` doesn't exist, `ip_addr` will be `null`
2. Never use `[[ ... ]] && ...` because it has a side effect of non-zero exit code, and in case `set -x` is on, it will abort the whole script. Always use `[[ ... ]] || ...` - just think it as a complete `if` statement.

Reply:
i tested the case when file is not existed, but not test the case when value not exist

Reply:
and there is no problem when creating and resuming the sandbox